src/puppetlabs/puppetdb/config.clj

```diff
@@ -156,7 +156,7 @@
    :subname     (format "file:%s;hsqldb.tx=mvcc;sql.syntax_pgs=true" (io/file vardir "db"))})
 
 (defn validate-db-settings
-  "Throws a {:type ::configuration-error :message m} exception
+  "Throws a {:type ::cli-error :message m} exception
```

Comment:
Fwiw I do think we should eventually update trapperkeeper to handle configuration errors as well, but I think it's out of the scope of this patch. We could have similar validation for configs as we do for cli's, in fact exactly the same for the most part (required items, parse-functions, validation functions).